src/store/navigation/reducer.js

```diff
@@ -0,0 +1,16 @@
+import * as types from './types';
+
+const initialState = {
+  toggleSidebar: true,
+};
+
+const navigation = (state = initialState, action) => {
+  switch (action.type) {
+    case types.TOGGLE_SIDEBAR:
+      return { ...state, toggleSidebar: !state.toggleSidebar };
```

Comment:
`toggleSidebar` is name of the action, not the state the sidebar is in.
what about
```
case types.TOGGLE_SIDEBAR:
      return { ...state, isSidebarVisible: !state.isSidebarVisible };
```